src/features/camera.md

```diff
@@ -0,0 +1,240 @@
+# Cameras
+
+{{#include ../include/links.md}}
+
+Cameras are what drives all rendering in Bevy. They are responsible for
+configuring what to draw, how to draw it, and where to draw it.
+
+You must have at least one camera entity, in order for anything to be displayed
+at all! If you forget to spawn a camera, you will get an empty black screen.
+
+In the simplest case, you can create a camera with the default settings. Just
+spawn an entity using [`Camera2dBundle`][bevy::Camera2dBundle] or
+[`Camera3dBundle`][bevy::Camera3dBundle]. It will simply draw all renderable
+entities that are [visible][cb::visibility].
+
+This page gives a general overview of cameras in Bevy. Also see the dedicated
+pages for [2D cameras][cb::camera-2d] and [3D cameras][cb::camera-3d].
+
+Practical advice: always create [marker components][cb::component-marker] for
+your camera entities, so that you can [query][cb::query] your cameras easily!
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:simple}}
+```
+
+## The Camera Transform
+
+Cameras have [transforms][cb::transform], which can be used to position or
+rotate the camera. This is how you move the camera around.
+
+For examples, see these [cookbook][chapter::cookbook] pages:
+ - [3D pan-orbit camera][cookbook::pan-orbit-camera], like in 3D editor apps
+
+If you are making a game, you should implement your own custom camera controls
+that feel appropriate to your game's genre and gameplay.
+
+### Zooming the camera
+
+You *could* use the transform's scale to scale/stretch the camera; however,
+don't confuse that with "zooming"! Typically, zooming should be done
+using the [projection](#projection).
+
+For orthographic projections, change the projection's scale. This way you can be
+confident about how exactly coordinate/units map to the screen, and avoid
+imprecision or visual artifacts. This also helps avoid scaling artifacts with 2D
+assets.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/d2.rs:zoom}}
+```
+
+For perspective projections, change the FOV. This achieves the desired 3D effect
+of zooming with a lens. Decrease the FOV to "zoom in" (make objects appear
+closer). Increase the FOV to "zoom out" (make objects appear further away,
+increase the stretching due to the perspective effect).
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/d3.rs:zoom}}
+```
+
+That said, in some cases (such as editor apps), using the transform scale
+might make sense. The effect it produces (stretching the content as it appears
+on screen) might feel better to control.
+
+## Projection
```

Comment:
Well my point was that these should be thought of as different use cases. FOV gives you the "camera lens zoom" effect, so in a game where you have a 3D camera and you want to zoom in and out, that's what you want. Scale just takes the image and makes it bigger. In an editor app, that might be what you want, you probably dont want to mess with FOV, as it would look weird.

I see that I might not have expressed it clearly. Do you have any ideas?